kitt/usr/share/asteroid-launcher/watchfaces/kitt.qml

```diff
@@ -0,0 +1,230 @@
+                style: timeDisplay.style
+                styleColor: timeDisplay.styleColor
+                Component.onCompleted: format()
+                function format(){
+                    var sec = watchFace.time.getSeconds();
+                    if (sec < 10) {
+                        sec = "0" + sec;
+                    }
+                    text = sec
+                }
+            }
+        }
+
+    }
+
+    Item {
+        id: chargingIndicatorArea
+        x: parent.width * 0.165
+        y: parent.width * 0.056
+        width: parent.width * 0.289
+        height: parent.width  * 0.115
+        Rectangle {
+            color: batteryChargePercentage.value < 50 ? 'red': 'green'
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.bottom: parent.bottom
+            width: batteryChargePercentage.value * parent.width/100
+            opacity: 0.5
+        }
+        Icon {
+            id: batteryIcon
+            name: "ios-battery-charging"
+            width:  parent.height * 0.6
+            height: width
+            anchors.right: parent.left
+            anchors.verticalCenter: parent.verticalCenter
+            opacity: 0.4
+            rotation: 270
+            visible: batteryIsCharging.value
+        }
+    }
+
+    Image {
+        id: compassRotateImage
+        x: parent.width * 0.037
+        y: parent.width * 0.65
+        width: parent.width * 0.36
+        height: width
+        sourceSize.height: parent.width
+        sourceSize.width: parent.width
+        fillMode: Image.PreserveAspectFit
+        source: "./kitt-compass.svg"
+    }
+
+    Image {
+        id: connectedImage
+
+        x: parent.width * 0.17
+        y: parent.width * 0.53
+
+        width: parent.width * 0.19
+        height: width * 0.308
+        sourceSize.height: width
+        sourceSize.width: height
+        fillMode: Image.PreserveAspectFit
+        source: "./kitt-connected.svg"
+        opacity: btStatus.connected ? 1 : btStatus.powered? 0.5 : 0.1
+    }
+
+    Image {
+        id: buttonImage
+        x: parent.width * 0.64
+        y: parent.width * 0.668
+        width: parent.width*0.298
+        height: parent.width * 0.32
+        sourceSize.height: height
+        sourceSize.width: width
+        fillMode: Image.PreserveAspectFit
+        source: "./kitt-button.svg"
+
+    }
+
+    Image {
+        id: foregroundImage
+
+        anchors.fill: parent
+        sourceSize.height: width
+        sourceSize.width: width
+        source: "./kitt-fg.svg"
+
+    }
+
+
+    BluetoothStatus {
+        id: btStatus
+    }
+
+    ContextProperty {
+        id: batteryChargePercentage
+        key: "Battery.ChargePercentage"
+        value: "100"
+        Component.onCompleted: batteryChargePercentage.subscribe()
+    }
+
+    ContextProperty {
+        id: batteryIsCharging
+        key: "Battery.IsCharging"
+        value: false
+    }
+
+
+    Connections {
```

Comment:
This is not needed

Reply:
Oh, that's right, forgot to remove the localeManager Connection.
Thanks, I'll remove it!